sdk/search/Azure.ResourceManager.Search/api/Azure.ResourceManager.Search.netstandard2.0.cs

```diff
@@ -226,6 +279,9 @@ public partial class SearchServiceNameAvailabilityResult
     public partial class SearchServicePatch : Azure.ResourceManager.Models.TrackedResourceData
     {
         public SearchServicePatch(Azure.Core.AzureLocation location) : base (default(Azure.Core.AzureLocation)) { }
+        public Azure.ResourceManager.Search.Models.DataPlaneAuthOptions AuthOptions { get { throw null; } set { } }
+        public bool? DisableLocalAuth { get { throw null; } set { } }
```

Comment:
Could we rename this property to `IsLocalAuthDisabled`?

Reply:
Did you do this rename? If you did, please generate the API file again to make sure the api file synced with the source code.